processing/src/main/java/io/druid/query/groupby/orderby/DefaultLimitSpec.java

```diff
@@ -255,6 +255,13 @@ public int compare(Row left, Row right)
       @Override
```

Comment:
but still `row` is not visible here `Comparator.comparing(row -> row.getRaw(column)`